AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Models/AuthCognitoTokens+Validation.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AWSPluginsCore
+
+/// Internal Helpers for managing session tokens
+extension AuthCognitoTokens {
+    
+    func areTokensExpiring(in seconds: TimeInterval? = nil) -> Bool {
```

Comment:
Do we need this value to be optional? Also add a comment for the function, mention that we consider the token to be expired if we cannot retrieve a valid expiration value from the tokens. 

```suggestion
    func areTokensExpiring(in seconds: TimeInterval = 0) -> Bool {
```